.checkov.baseline

```diff
@@ -1,5 +1,19 @@
 {
     "failed_checks": [
+        {
+            "file": "/cdn/main.tf",
+            "findings": [
+                {
+                    "resource": "module.extensions-staging.module.cdn.aws_cloudfront_distribution.standard",
+                    "check_ids": [
+                        "CKV_AWS_305",
+                        "CKV_AWS_310",
+                        "CKV2_AWS_32",
+                        "CKV2_AWS_47"
```

Comment:
This is not a good way to address Checkov's findings.

We should not be adding new things to the Checkov baseline file.

It is only intended to allow easy adoption of checkov by temporarily silencing warnings about existing misconfigurations whilst still alerting about new ones.

The existing misconfigurations should be addressed as time allows and the baseline file will end up empty. We have a ticket for that.

If it alerts about something we genuinely need Checkov to ignore, the checks can be skipped in context in the files which should be accompanied by a comment explaining why it is skipped for our future selves benefit.

Reply:
None of these are relavant to our configuration of CloudFront. I didnt know how else to skip the check

Reply:
moving these